CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
-## 0.10.5-dev3
+## 0.10.5
 
 ### Enhancements
 * Create new CI Pipelines
   - Checking text, xml, email, and html doc tests against the library installed without extras
   - Checking each library extra against their respective tests
-* `partition` raises and error and tells the user to install the appropriate extra if a filetype
+* `partition` raises an error and tells the user to install the appropriate extra if a filetype
   is detected that is missing dependencies.
 * Add custom errors to ingest
+* Bump `unstructured-ingest` to 0.5.15
```

Comment:
please indicate unstructured-inference changes in sub-bullets.
e.g.:

https://github.com/Unstructured-IO/unstructured/blob/106ee965a6d072fa399e255a98fd761476e490f1/CHANGELOG.md?plain=1#L37-L39